### PR TITLE
Issue-16 get posts of current author

### PR DESCRIPTION
**Pull request by PENGYUXIONG**

### CODE REVIEW

File: frontend/src/pages/myPostsPage.jsx [hunk 1]
```diff
@@ -58,15 +71,16 @@ class myPostsPage extends React.Component {
...
-                extra={
-                  (
-                    <img
-                      width={272}
-                      alt="logo"
-                      src="https://gw.alipayobjects.com/zos/rmsportal/mqaQswcyDLcXyDKnZfES.png"
-                    />
-                  )
-                }
+                // display picture
```
Comment (AnkushSharma2698) [inline frontend/src/pages/myPostsPage.jsx]: remove commented code if not needed

File: backend/api/services/authorServices.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
 from rest_framework import status
 from rest_framework.response import Response
 
+import sys
```
Comment (AnkushSharma2698) [inline backend/api/services/authorServices.py]: remove if not used

File: backend/api/views/simplePostView.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 from ..services.postServices import postServices
-
+import sys
```
Comment (AnkushSharma2698) [inline backend/api/views/simplePostView.py]: remove if not used

File: backend/api/views/simplePostView.py [hunk 4]
```diff
@@ -38,14 +38,15 @@ def createNewPost(request, author_id):
 # pageNum, optional, int
 #####################################
 def getPostsByAuthorId(request, author_id, pageNum = 0):
-  request.data['author_id'] = author_id
+  if 'pageNum' in request.data.keys():
+    pageNum = int(request.data.dict()['pageNum'])
   res = postServices.getPostByAuthorId(request, author_id)
   if res.status_code == 404:
     return res
   pagedRes = postServices.getPaginatedPosts(res, pageNum).data
```
Comment (AnkushSharma2698) [inline backend/api/views/simplePostView.py:46]: i think this is fine for now but we should move to using the pagination class provided by django, it error handles for us

File: frontend/src/pages/myPostsPage.jsx [hunk 5]
```diff
@@ -30,9 +22,30 @@ const IconText = ({ icon, text }) => (
...
+    listData = []
+    this.addPostsIntoList(this.props.post);
+  }
+
+  addPostsIntoList = (posts) => {
+    for (let i = 0; i < posts.length; i++) {
+      listData.push({
+        href: `${posts[i]['id']}`,
+        title: `${posts[i]['title']}`,
+        avatar: 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png',
```
Comment (AnkushSharma2698) [inline frontend/src/pages/myPostsPage.jsx:41]: is avatar needed? I dont think profile handles user images

**Review by AnkushSharma2698 — COMMENTED** [review]
Overall looks great! Will approve once comments addressed

File: frontend/src/pages/myPostsPage.jsx (hunk 5 above)
Comment (PENGYUXIONG) [reply frontend/src/pages/myPostsPage.jsx:41]: I am leaving there for UI beautify mostly, simply plain texts looks ugly

File: backend/api/views/simplePostView.py (hunk 4 above)
Comment (PENGYUXIONG) [reply backend/api/views/simplePostView.py:46]: My struggle is mostly from how to get the page number. It is not known yet whether it comes from the body or param.

**Pull request merged**